conda_build/license_family.py

```diff
@@ -14,11 +13,11 @@
 GPL
 BSD
 MIT
-APACHE
+Apache
 PSF
-PUBLICDOMAIN
-PROPRIETARY
-OTHER
+Public-Domain
```

Comment:
> I generally advise you to not set license family manually. If you have strong reason to, we should talk more.

We started doing it at conda-forge after multiple requests to do so. Are you saying we should stop doing this now? TBH I don't care either way. That's just going to be a lot of recipes to fix and people's behavior to correct.